src/build-a-bull/src/features/vote/index.tsx

```diff
@@ -77,16 +75,18 @@ function Vote({ sort: sortProp = 'none' }: { sort?: 'ascending' | 'descending' |
   const hasVoteEnded = !votingRoundGlobalState ? false : getHasVoteEnded(votingRoundGlobalState)
   const isVoteCreator = votingRoundMetadata?.created.by === activeAddress ? true : false
 
-  const canVote = hasVoteStarted && !hasVoteEnded && allowedToVote
+  const canVote = hasVoteStarted && !hasVoteEnded
   const hasVoted = voterVotes !== undefined ? true : false
   const hasClosed = votingRoundGlobalState && votingRoundGlobalState.close_time !== undefined ? true : false
 
+
+  const totalVotes = useMemo(()=>votingRoundResults ? votingRoundResults.reduce((c, r)=>c + r.count, 0) : 0, [votingRoundResults])
```

Comment:
did you make sure that this works well now that you have 2 options per question?

Reply:
Just saw the same bug locally, need to filter for Yes options